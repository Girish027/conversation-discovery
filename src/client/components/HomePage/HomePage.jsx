import React from 'react';
import Sidebar from 'components/Sidebar';
import ContentPanel from 'components/ContentPanel';

function HomePage() {
  return (
    <main className='main-container'>
      <div className='sidebar'>
        <Sidebar />
      </div>
      <div className='content'>
        <ContentPanel />
      </div>
    </main>
  );
}

export default HomePage;
