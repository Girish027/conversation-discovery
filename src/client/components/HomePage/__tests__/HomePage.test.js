import React from 'react';
import { shallow } from 'enzyme';
import toJSON from 'enzyme-to-json';
import HomePage from 'components/HomePage';

describe('<HomePage />', () => {
  let wrapper;

  describe('Creating an instance', () => {
    test('should exist', () => {
      wrapper = shallow(<HomePage />);
      expect(wrapper.exists()).toBe(true);
    });
  });

  describe('Snapshots', () => {
    test('renders correctly with Sidebar and content', () => {
      wrapper = shallow(<HomePage />);
      expect(toJSON(wrapper)).toMatchSnapshot();
    });
  });
});
