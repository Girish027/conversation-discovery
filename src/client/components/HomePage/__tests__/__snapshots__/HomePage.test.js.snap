// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<HomePage /> Snapshots renders correctly with Sidebar and content 1`] = `
<main
  className="main-container"
>
  <div
    className="sidebar"
  >
    <Sidebar />
  </div>
  <div
    className="content"
  >
    <Connect(ContentPanel) />
  </div>
</main>
`;
